incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/config/OpenapiServerCompositeBindingAdapter.java

```diff
@@ -0,0 +1,440 @@
+import java.util.regex.Pattern;
+
+import org.agrona.collections.Object2ObjectHashMap;
+
+import io.aklivity.zilla.runtime.binding.http.config.HttpAuthorizationConfig;
+import io.aklivity.zilla.runtime.binding.http.config.HttpConditionConfig;
+import io.aklivity.zilla.runtime.binding.http.config.HttpOptionsConfig;
+import io.aklivity.zilla.runtime.binding.http.config.HttpOptionsConfigBuilder;
+import io.aklivity.zilla.runtime.binding.http.config.HttpRequestConfig;
+import io.aklivity.zilla.runtime.binding.http.config.HttpRequestConfigBuilder;
+import io.aklivity.zilla.runtime.binding.openapi.config.OpenapiConfig;
+import io.aklivity.zilla.runtime.binding.openapi.config.OpenapiOptionsConfig;
+import io.aklivity.zilla.runtime.binding.openapi.internal.OpenapiBinding;
+import io.aklivity.zilla.runtime.binding.openapi.internal.model.MediaType;
+import io.aklivity.zilla.runtime.binding.openapi.internal.model.OpenApi;
+import io.aklivity.zilla.runtime.binding.openapi.internal.model.Operation;
+import io.aklivity.zilla.runtime.binding.openapi.internal.model.Parameter;
+import io.aklivity.zilla.runtime.binding.openapi.internal.model.Server;
+import io.aklivity.zilla.runtime.binding.openapi.internal.view.PathView;
+import io.aklivity.zilla.runtime.binding.openapi.internal.view.SchemaView;
+import io.aklivity.zilla.runtime.binding.openapi.internal.view.ServerView;
+import io.aklivity.zilla.runtime.binding.tcp.config.TcpConditionConfig;
+import io.aklivity.zilla.runtime.binding.tcp.config.TcpOptionsConfig;
+import io.aklivity.zilla.runtime.binding.tls.config.TlsOptionsConfig;
+import io.aklivity.zilla.runtime.engine.config.BindingConfig;
+import io.aklivity.zilla.runtime.engine.config.BindingConfigBuilder;
+import io.aklivity.zilla.runtime.engine.config.CompositeBindingAdapterSpi;
+import io.aklivity.zilla.runtime.engine.config.GuardedConfigBuilder;
+import io.aklivity.zilla.runtime.engine.config.ModelConfig;
+import io.aklivity.zilla.runtime.engine.config.NamespaceConfigBuilder;
+import io.aklivity.zilla.runtime.engine.config.RouteConfigBuilder;
+import io.aklivity.zilla.runtime.model.core.config.IntegerModelConfig;
+import io.aklivity.zilla.runtime.model.core.config.StringModelConfig;
+import io.aklivity.zilla.runtime.model.json.config.JsonModelConfig;
+
+public final class OpenapiServerCompositeBindingAdapter implements CompositeBindingAdapterSpi
+{
+    private static final String INLINE_CATALOG_NAME = "catalog0";
+    private static final Pattern JSON_CONTENT_TYPE = Pattern.compile("^application/(?:.+\\+)?json$");
+
+    private final Matcher jsonContentType = JSON_CONTENT_TYPE.matcher("");
+    private final Map<String, ModelConfig> models = Map.of(
+        "string", StringModelConfig.builder().build(),
+        "integer", IntegerModelConfig.builder().build()
+    );
+
+    @Override
+    public String type()
+    {
+        return OpenapiBinding.NAME;
+    }
+
+    public OpenapiServerCompositeBindingAdapter()
+    {
+    }
```

Comment:
No need to define a public no-args constructor, as it is implicitly defined by default if no other constructors are defined.